### PR TITLE
Add MySQL to the AWS Services docs

### PR DESCRIPTION
**Pull request by gcapizzi**
# Which other branches do you want a technical writer to cherry-pick this PR to (if any)?

None, this is TAP 1.7.0 only.

### CODE REVIEW

File: aws-services/reference/package-values.hbs.md [hunk 1]
```diff
@@ -28,3 +28,20 @@ The following table lists configuration that applies to the `postgresql` service
...
+| mysql.enabled                                    | `false`               | boolean | Enables the MySQL service class                                                                                                                                                                                                                                                   |
+| mysql.infrastructure.security_groups             | _n/a_                 | array   | Security groups for your MySQL databases to belong to                                                                                                                                                                                                                             |
+| mysql.infrastructure.subnet_group.name           | `""`                  | string  | Subnet group for your MySQL databases                                                                                                                                                                                                                                             |
```
Comment (ChunyiLyu) [inline aws-services/reference/package-values.hbs.md:40]: I think it's worth adding description for `security_groups` and `subnet_group` so we are not leaving them blank.

File: aws-services/reference/supported-services.hbs.md [hunk 2]
```diff
@@ -2,7 +2,9 @@
 
 This topic lists the services that are supported by the AWS Services package.
 
-## <a id="postgresql"></a> RDS PostgreSQL
+In Tanzu Application Platform v1.7, the AWS Services package only supports the following AWS services:
```
Comment (ChunyiLyu) [inline aws-services/reference/supported-services.hbs.md:5]: Nitpick I would not say `only` here. "the AWS Services package supports the following AWS services" should be sufficient.

File: aws-services/reference/supported-topologies.hbs.md [hunk 3]
```diff
@@ -41,16 +41,28 @@ For instructions for these tasks, see the
...
+    ```yaml
+    mysql:
+      enabled: true
+      region: "REGION"
+      infrastructure:
+        subnet_group:
+          name: "SUBNET-GROUP-NAME"
+        security_groups:
+          - id: "SECURITY-GROUP-ID"
+    ```
```
Comment (ChunyiLyu) [inline aws-services/reference/supported-topologies.hbs.md:65]: I suppose that if one of them, either MySQL or Postgres, is disabled earlier at installation, you don't need to provide values for it?

File: aws-services/reference/package-values.hbs.md [hunk 4]
```diff
@@ -28,3 +28,20 @@ The following table lists configuration that applies to the `postgresql` service
...
+| mysql.instance_configuration.publicly_accessible | `false`               | boolean | Controls whether your instances are publicly accessible                                                                                                                                                                                                                                |
+| mysql.instance_configuration.skip_final_snapshot | `false`               | boolean | Determines whether a final snapshot is created before the instance is deleted. If you specify true, no snapshot is created. If you specify false, a snapshot called final-snapshot-INSTANCE-NAME is created before the instance is deleted.                                            |
+| mysql.provider_config_ref.name                   | `default`             | string  | Name of your ProviderConfig for the `mysql` service                                                                                                                                                                                                                               |
```
Comment (ChunyiLyu) [inline aws-services/reference/package-values.hbs.md:46]: Do we have both postgres and mysql provider config name as `default`? Doesn't that run into conflict?

File: aws-services/reference/package-values.hbs.md (hunk 1 above)
Comment (gcapizzi) [reply aws-services/reference/package-values.hbs.md:40]: They have one 🤔

File: aws-services/reference/package-values.hbs.md (hunk 4 above)
Comment (gcapizzi) [reply aws-services/reference/package-values.hbs.md:46]: Given this is provided by the user, there's no issue - they'll just both use the same `ProviderConfig`. I think `default` gets used anyway as it's also Crossplane's default, but it's the best value we can use for our own default too 🤷

File: aws-services/reference/supported-services.hbs.md (hunk 2 above)
Comment (gcapizzi) [reply aws-services/reference/supported-services.hbs.md]: I agree - fixed!

File: aws-services/reference/supported-topologies.hbs.md (hunk 3 above)
Comment (gcapizzi) [reply aws-services/reference/supported-topologies.hbs.md:65]: they're not two separate phases - you provide values at installation, and that's how you enable services

File: aws-services/reference/package-values.hbs.md (hunk 1 above)
Comment (ChunyiLyu) [reply aws-services/reference/package-values.hbs.md:40]: My bad. Misread the column

**Review by ChunyiLyu — APPROVED** [review]

**Pull request merged**